database/README.md

```diff
@@ -11,12 +11,24 @@
 
 <br />
 
-# Knex Migrations/Seeds
```

Comment:
I think this heading makes sense cause of the two sub-sections that follow, so maybe we can keep it?

Reply:
I still think we want to keep this

Reply:
better now?